spec/functions/ip_to_cron_spec.rb

```diff
@@ -42,15 +40,13 @@
 
   describe 'when runinterval is 30 minutes' do
     let(:runinterval) { 1800 }
+    let(:facts) { { networking: { ip: '10.0.0.1' } } }
 
-    before do
-      allow(scope).to receive(:lookupvar).with('ipaddress').and_return('10.0.0.1')
-    end
     it 'returns \'*\' for the hour' do
```

Comment:
Shouldn't we be able to verify this with `it { is_expected.to run.with_params(runinterval).and_return(['*', [30]]) }` since it should be predictable based on the IP?

Reply:
maybe?  There are other improvements that could be made, but my primary goal was to convert the function (and get some documentation in for it).